src/components/header/_macro.spec.js

```diff
@@ -567,110 +392,50 @@ describe('macro: header', () => {
-    });
-
-    it('has the provided custom class', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
-
-      expect($('.ons-header-service-nav--main').hasClass('custom-class')).toBe(true);
-    });
-
-    it('has the provided `aria-label` for the list for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
-
-      expect($('.ons-header-service-nav--mobile').attr('aria-label')).toBe('Services menu');
-    });
-
-    it('has the link text for each list item for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE));
-
-      const values = mapAll($('.ons-header-service-nav--mobile .ons-header-service-nav__link'), (node) => node.text().trim());
-      expect(values).toEqual(['Title 1', 'Title 2', 'Title 3']);
-    });
-
-    it('has the link href for each list item for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE));
-
-      const values = mapAll($('.ons-header-service-nav--mobile .ons-header-service-nav__link'), (node) => node.attr('href'));
-      expect(values).toEqual(['#1', '#2', '#3']);
-    });
-
-    it('renders a button with expected parameters', () => {
       const faker = templateFaker();
       const buttonSpy = faker.spy('button', { suppressOutput: true });
-
       faker.renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE);
 
-      expect(buttonSpy.occurrences).toContainEqual({
-        text: 'Menu',
-        classes: 'ons-u-d-no ons-js-toggle-services',
-        type: 'button',
-        variants: ['mobile', 'text-link'],
-        attributes: {
-          'aria-label': 'Toggle services menu',
-          'aria-controls': 'service-links',
-          'aria-expanded': 'false',
-        },
+      test('THEN: renders with correct display class on desktop', () => {
+        expect($('.ons-header__links .ons-grid__col').hasClass('ons-u-d-no@xxs@m')).toBe(true);
+      });
+      test('THEN: renders button on mobile', () => {
+        expect(buttonSpy.occurrences).toContainEqual({
+          text: 'Menu',
+          classes: 'ons-u-d-no ons-js-toggle-services',
+          type: 'button',
+          variants: ['mobile', 'text-link'],
+          attributes: {
+            'aria-label': 'Toggle services menu',
+            'aria-controls': 'service-links',
+            'aria-expanded': 'false',
+          },
+        });
       });
     });
-
-    it('renders a button with correct variant if `internal`', () => {
+    describe('WHEN: one item is provided to the itemsList parameter', () => {
+      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
+      test('THEN: renders without multilink display class on desktop', () => {
+        expect($('.ons-header__links .ons-grid__col').hasClass('ons-u-d-no@xxs@')).toBe(false);
```

Comment:
The second breakpoint is missing off the `ons-u-d-no@xxs@` class

Reply:
This is actually the case in the original code as well - not entirely sure what the desired second break point is

Reply:
I think I see what its doing here looking at the tests above it, is its supposed to be checking for the existence of either the `ons-u-d-no@xxs@m` or `ons-u-d-no@xxs@xs` classes. I think we should probably just do that explicitly because its a bit confusing like this. Especially now the tests are a bit further apart. There is also the possibility that this isn't actually working and the `hasClass` function is looking specifically for `ons-u-d-no@xxs@` which it will never find but I haven't checked that in the docs

Reply:
Its not meant to find either. Thats the test. If one service link/language is in the masthead then it doesn't change to a menu button on small screens because it doesn't have either of those classes. Then if there are two or 3 or more links then it will change to a button at different break points. The `ons-u-d-no` utility class sets display to none for the button to appear instead. So what we need to do is to change this test to test that either class explicitly doesn't exist not `ons-u-d-no@xxs@`.

Reply:
I couldn't work out if it was supposed to be M or XS was the issue - but Alessio helped find it is supposed to be M.

Reply:
It needs to be both. We need to update the test to check that both `ons-u-d-no@xxs@m` and `ons-u-d-no@xxs@xs` do not exist in the class list

Reply:
btw this has been added!